sdk/aqueduct/utils/function_packaging.py

```diff
@@ -207,7 +207,7 @@ def _package_files_and_requirements(
 
         elif os.path.exists(REQUIREMENTS_FILE):
             # There exists a workflow-level requirements file (need to reside in the same directory as the function).
-            logger().debug(
+            logger().info(
```

Comment:
Actually, we should just delete this log statement altogether. We no longer infer package dependencies.